docs/integrations.rst

```diff
@@ -268,6 +268,10 @@ OpenVAS CSV
 -----------
 Import OpenVAS Scan in CSV format. Export as CSV Results on OpenVAS.
 
+OssIndex Devaudit
+-----------------
+OSSIndex Devaudit Scan output can be imported in JSON format.

```

Comment:
can add a link to the ossindex devaudit site or something like that?

Reply:
Done in second commit.